packages/help-center/src/hooks/use-launchpad.tsx

```diff
@@ -0,0 +1,52 @@
+import apiFetch from '@wordpress/api-fetch';
```

Comment:
The approach I take in this hook - including the use of react-query, and the conditional use of wpcomRequest and apiFetch is patterned from existing approaches used in help center. See this file for example: 

https://github.com/Automattic/wp-calypso/blob/trunk/packages/help-center/src/hooks/use-help-search-query.ts